src/components/BotBuilder/BotBuilder.js

```diff
@@ -487,15 +450,22 @@ class BotBuilder extends React.Component {
 BotBuilder.propTypes = {
   templates: PropTypes.array,
   actions: PropTypes.object,
+  accessToken: PropTypes.string,
 };
 
+function mapStateToProps(store) {
```

Comment:
We do not need accessToken

Reply:
@fragm3 Inside getChatbots() chatbots should be fetched only if the user is logged in. I guess instead of using cookies to get logged in status. It will be better to use the access token.

Reply:
@harsh-jindal Okay, let's pass this one